VSDiagnostics/VSDiagnostics/VSDiagnostics.Test/Tests/General/UseAliasesInsteadOfConcreteTypeTests.cs

```diff
@@ -1596,13 +1596,13 @@ class Foo
     {
         static void Main()
         {
-            string bar = ""test"";
+            Single bar = ""test"";
```

Comment:
I don't get it. Why is this a fix in the first place? Shouldn't it have no warning?